src/components/documents/documentsNycCerts.tsx

```diff
@@ -0,0 +1,121 @@
+import { Document, Tag } from "../documents/types";
+import { uriToAction } from "../documents/utils";
+
+const makeNycDocument = ({
+  title,
+  oaFilename,
+  imgFilename
+}: {
+  title: string;
+  oaFilename: string;
+  imgFilename: string;
+}): Document => ({
+  title,
+  documents: [
+    {
+      uri: uriToAction({
+        uri: `https://document-storage.oa.gov.sg/national-youth-council_${oaFilename}`,
+        permittedActions: ["VIEW"],
+        redirect: "https://dev.verify.gov.sg/verify"
+      }),
+      kind: "verifiable",
```

Comment:
Small change here since the documents are signed with a DID:

```suggestion
      kind: "did",
```